test/core/tsi/alts/handshaker/alts_concurrent_connectivity_test.cc

```diff
@@ -334,9 +334,17 @@ class FakeTcpServer {
     CLOSE_SOCKET,
   };
 
-  FakeTcpServer(
+  enum AcceptMode {
```

Comment:
I think the C++ style guide suggests using enum classes https://google.github.io/styleguide/cppguide.html#Enumerator_Names
enum class AcceptMode { kWaitForClientToSendFirstBytes, kEagerlySendSettings };

Reply:
done